versions.def

```diff
@@ -71,11 +71,11 @@ HTTPD_VERSION		:= 2.4.53
 
 IMGK_VERSION		:= 6.7.7-2
 
-JDK_SUB_TREE		:= 8u
-JDK_UPDATE		:= 172
-JDK_BUILD		:= b01
-JDK_TAG			:= jdk8u$(JDK_UPDATE)-$(JDK_BUILD)
-JDK_VERSION		:= 1.8.0u$(JDK_UPDATE)$(JDK_BUILD)
+JDK_SUB_TREE		:= 17
+JDK_UPDATE		:= 17.0
+JDK_BUILD		:= 2
```

Comment:
any reason we are not using 17.0.3.1 ?

Reply:
when we pickup this task, 17.0.3.1 not yet released.